src/pages/workspaces/workspace/notebooks/modals/CloudEnvironmentModal.js

```diff
@@ -63,13 +64,10 @@ export const CloudEnvironmentModal = ({
       setViewMode(undefined)
       onDismiss()
     },
-    onSuccess: _.flow(
-      withErrorReporting('Error creating cloud compute'),
-      Utils.withBusyState(setBusy)
-    )(async () => {
+    onSuccess: () => {
       setViewMode(undefined)
-      await refreshRuntimes(true)
-    })
+      onDismiss()
+    }
```

Comment:
Could you help me understand the following?

2. Why are we removing the error reporting? What happens if `onDismiss()` throws an error?
2. Why are we no longer setting `busy` to `true`? 

Same questions for below.

Reply:
So this error reporting is actually duplicative. The modal itself has its own error reporting wrapper, and this was interfering.

We are not setting busy to true here as this is no longer an async blocking call. This was also duplicative, and you'll notice that any busy spinner + overlay is handled by the inner modal.

This was part of an investigation to correctly display the spinner and any errors associated with async actions.

Reply:
Sounds great. Thank you.